src/server/actions/delPlanMedLege.ts

```diff
@@ -19,7 +20,7 @@ export async function delPlanMedLegeServerAction(
     await simulateBackendDelay();
 
     return {
-      deltMedLegeTidspunkt: new Date(),
+      deltMedLegeTidspunkt: now().toISOString(),
```

Comment:
Kunne evnt vurdert å hatt noe util for å slippe å gjøre .toISOString() manuelt her og der, i tilfelle det skulle glippe.